discord/bot.go

```diff
@@ -667,10 +675,22 @@ func (guild *GuildState) handleMessageCreate(s *discordgo.Session, m *discordgo.
 						}
 					}
 
+					guild.PrivateStateMsg.privateChannelID = privateChannelIDTemp;
+
 					guild.handleGameStartMessage(s, m, room, region, initialTracking)
+					guild.PrivateStateMsg.idUsernameMap = make(map[string]string)
+					guild.PrivateStateMsg.printedUsers = make([]string, 0)
+					guild.PrivateStateMsg.currentUserID = ""
+					guild.PrivateStateMsg.message = nil;
+					guild.createPrivateMapMessage(s, m, initialTracking)
 					break
 				case End:
 					guild.handleGameEndMessage(s)
+					var pMessage = guild.PrivateStateMsg.message;
```

Comment:
I would use your PrivateStateMsg.Exists() here (or rather, in the if check below) instead :)

Reply:
Good catch haha